cmd/profile.go

```diff
@@ -24,6 +24,12 @@ var profilecmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(profilecmd)
 	profilecmd.Flags().StringVar(&profileOptions.GRPC, "gRPC", "", "use gRPC")
-	profilecmd.Flags().StringVar(&profileOptions.Namespace, "namespace", "", "Filter using namespace")
+	profilecmd.Flags().StringVarP(&profileOptions.Namespace, "namespace", "n", "", "Filter using namespace")
 	profilecmd.Flags().StringVar(&profileOptions.Pod, "pod", "", "Filter using Pod name")
+	// logCmd.Flags().StringVar(&logOptions.LogType, "logType", "", "Log type you want (Eg:ContainerLog/HostLog) ")
+	// logCmd.Flags().StringVar(&logOptions.ContainerName, "container", "", "name of the container ")
+	profilecmd.Flags().StringVar(&logOptions.Resource, "resource", "", "command used by the user")
+	profilecmd.Flags().StringVar(&logOptions.Source, "source", "", "binary used by the system ")
+	profilecmd.Flags().Uint32Var(&logOptions.Limit, "limit", 0, "number of logs you want to see")
```

Comment:
I didn't understand the usecase of limit. Why would user say I want to limit it to max 100 logs or something?

If not required, we can remove this option and reduce the code complexity.